Content/Passives/Ranged/Masteries/MarksmanMastery.cs

```diff
@@ -0,0 +1,94 @@
+﻿using PathOfTerraria.Common.Systems.PassiveTreeSystem;
+using PathOfTerraria.Utilities;
+using ReLogic.Content;
+using System.Collections.Generic;
+
+namespace PathOfTerraria.Content.Passives;
+
+internal class MarksmanMastery : Passive
+{
+	internal class MarkedNPC : GlobalNPC
+	{
+		private static readonly Asset<Texture2D> Icon = ModContent.Request<Texture2D>("PathOfTerraria/Assets/Misc/VFX/MarkedIcon");
+
+		private static bool _markedDamage = false;
```

Comment:
I once again forgot that we should be using this attribute for any temporary overrides. May need to go through usages of `ValueOverride` so far to distribute it further. It is very unlikely for a bug to arise without it, but the recently resolved accessory problem was exactly caused by a lack of threadlocality on TML's side.
```suggestion
		[ThreadStatic]
		private static bool _markedDamage = false;
```